metrics/metrics.go

```diff
@@ -15,6 +15,53 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+type DKGStatus uint8
+type ReshareStatus uint8
+
+const (
+	DKGNotStarted DKGStatus = iota
+	DKGInProgress
+	DKGWaiting
+	DKGReady
+)
+
+const (
+	ReshareIdle ReshareStatus = iota
+	ReshareWaiting
+	ReshareInProgess
+)
+
+const Unknown = "unknown"
+const waiting = "waiting"
+
+func (s DKGStatus) String() string {
+	switch s {
+	case DKGNotStarted:
+		return "not_started"
+	case DKGInProgress:
+		return "in_progess"
+	case DKGWaiting:
+		return waiting
+	case DKGReady:
+		return "ready"
+	default:
+		return Unknown
+	}
```

Comment:
why are some constants and others not?

Reply:
I had to add them as constants because the linter complained about using the same string twice.